style.css

```diff
@@ -220,34 +290,342 @@ h2 {
   padding: 0 20px 0 20px;
   border-radius: 6px;
   background-color: #ebebff;
+  transition-property: background-color font-size color;
+  transition-duration: 200ms 500ms 200ms;
 }
 
-/* End of projects section */
+.lng-list div {
+  background-color: #f7f7f9;
+  justify-content: center;
+  align-self: stretch;
+  border-radius: 10px;
+  margin: 0;
+}
 
-@media only screen and (max-width: 600px) {
-  .container {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-content: center;
-    min-height: 100vh;
-    width: 100%;
-    padding: 0;
-    margin: 0;
+.lng li:hover {
+  font-size: 1.2em;
+  background-color: #7a869a;
+  color: #ebebff;
+}
+
+.btn {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  border: solid #396df2 2px;
+  border-radius: 12px;
+  padding: 12px 12px;
+  background-color: #fff;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 20px 0 20px 0;
+}
+
+.btn:hover {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  border: solid #396df2 2px;
+  border-radius: 12px;
+  padding: 12px 12px;
+  background-color: #6070ff;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  margin: 20px 0 20px 0;
```

Comment:
- [ ] [REQUIRED] I think it would be better if you remove the margin from the hover effect of the buttons. It makes the button acting different to the Figma template.

![hover](https://user-images.githubusercontent.com/60979926/165877799-541ce94a-d023-4281-8c08-ee3eaaa5ae5d.gif)

> **Note:** Always try to stick to figma design as possible.